src/util/range.h

```diff
@@ -317,7 +317,7 @@ struct zip_iteratort
   }
 
   /// Postincrement operator
-  const zip_iteratort operator++(int)
+  zip_iteratort operator++(int)
```

Comment:
@thomasspriggs Does this change look ok?

Reply:
`const` means nothing here but @thomasspriggs points out that this is done throughout the file. I'll remove this commit and create a separate PR to clean up the whole file.

Reply:
Ack, I just wanted to make sure @thomasspriggs is aware. Thank you for working this out together!